<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Jiacheng Miao</title>
  
  <meta name="author" content="Jiacheng Miao">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/icon.png">
</head>
  
 <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-XV4HQPS3S5"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-XV4HQPS3S5');
</script>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Jiacheng Miao</name>
              </p>
              <p>I'm a second-year Ph.D. student in Biomedical Data Science at UW-Madison, where I am very fortunate to work with Prof. <a href="http://qlu-lab.org">Qiongshi Lu</a>. and Prof. <a href="https://sites.google.com/site/laurenschmtz/">Lauren Schmitz</a>. I got my B.S. in Statistics from Nanjing University in 2019. 
              </p>
              <p style="text-align:center">
                <a href="mailto:jmiao24@wisc.edu">Email</a> &nbsp/&nbsp
                <a href="https://twitter.com/Jiacheng_Miao">Twitter</a> &nbsp/&nbsp
                <a href="https://github.com/jmiao24/">Github</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/avatar.JPG"><img style="width:80%;max-width:50%" alt="profile photo" src="images/avatar.JPG" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Research</heading>
              <p>
                My research interests lie in statistical genetics, population genetics and Bayesian methods. Currently, I am working on developing generic tools for statistical genetics especially gene-environment interactions. Below are my publications:
              </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>


        </tr>
      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/X-Wing.png" alt="blind-date" width="160" height="160">
          </td>
          <td width="75%" valign="middle">
            <strong>[5]</strong> <a href="https://www.biorxiv.org/content/10.1101/2022.05.26.493528v1">
             <papertitle>Quantifying portable genetic effects and improving cross-ancestry genetic prediction with GWAS summary statistics   </papertitle>
            </a>
            <br>
            <strong>Miao J.*</strong>, Guo H.*, Song G., Zhao Z., Hou L.<sup>&dagger;</sup>, Lu Q.<sup>&dagger;</sup> (2022). 
            <br>
            <em>Submitted. (preprint available on bioRxiv)</em>
            <br>
            <a href="https://www.biorxiv.org/content/10.1101/2022.05.26.493528v1">Preprint</a>
    /
            <a href="https://github.com/qlu-lab/X-Wing">Software</a>
            <p></p>
            <p>We propose a cross-population genetic risk prediction framework that (1) quantifies cross-population local genetic correlation
              (2) incorporates it as an annotation into a Bayesian framework which amplifies correlated SNP effects between populations through annotation-dependent statistical shrinkage 
              (3) combines multiple population-specific PRS to further improve prediction accuracy while using GWAS summary data alone as input.
               </p>
          </td>
        </tr>
        
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/vQTL.jpeg" alt="blind-date" width="160" height="160">
            </td>
            <td width="75%" valign="middle">
              <strong>[4]</strong> <papertitle>Identifying genetic loci associated with complex trait variability. </papertitle>
              </a>
              <br>
              <strong>Miao J.</strong>, Lu Q. (2022). 
              <br>
              <em>Handbook of Statistical Bioinformatics (2nd Edition). Springer, in press. (Book Chapter)  </em>
              <br>
              <p>
              </p>
            </td>
          </tr>

        </tr>
      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/QUAIL.jpeg" alt="blind-date" width="160" height="160">
          </td>
          <td width="75%" valign="middle">
            <strong>[3]</strong><a href="https://www.biorxiv.org/content/10.1101/2021.04.14.439847v1">
              <papertitle>A quantile integral linear model to quantify genetic effects on phenotypic variability.   </papertitle>
            </a>
            <br>
            <strong>Miao J.</strong>, Lin Y., Wu Y., Zheng B., Schmitz L., Fletcher J., Lu Q. (2021). 
            <br>
            <em>Submitted. (preprint available on bioRxiv)</em>
            <br>
            <a href="https://www.biorxiv.org/content/10.1101/2021.04.14.439847v1">Preprint</a>
    /
            <a href="https://github.com/qlu-lab/QUAIL">Software</a>
            <p>
              <ul>
                - Winner of <font color="red"><strong>Distinguished Student Paper Award</strong></font> from the Section on Statistical Genomics and Genetics of the American Statistical Association (ASA) 2022.</li>
                <br> - Winner of <font color="red"><strong>Reviews's Choice Award</strong></font> from American Society of Human Genetics Meeting (ASHG) Meeting 2021.</li>
              </ul>
            </p>
            <p>We propose a unified statistical framework for estimating genetic effects on the variability of quantitative traits that (1) constructs a quantile integral phenotype which aggregates 
              information from all quantile levels, and only requires fitting two linear regressions per SNP in genome-wide analysis. (2) addresses some limitations of current vQTL methods, 
              including a lack of robustness to non-Gaussian phenotypes and confounding effects on both trait levels and trait variability.
               </p>
          </td>
        </tr>
          
        </tr>
      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/epigen1.jpeg" alt="blind-date" width="160" height="160">
          </td>
          <td width="75%" valign="middle">
            <strong>[2]</strong><a href="https://www.tandfonline.com/doi/abs/10.1080/15592294.2021.1939479?journalCode=kepi20">
              <papertitle>The socioeconomic gradient in epigenetic aging clocks: evidence from the Multi-ethnic Study of Atherosclerosis and the Health and Retirement Study.  </papertitle>
            </a>
            <br>
            Schmitz L., Zhao W., Ratliff S., Goodwin J., <strong>Miao J.</strong>, Lu Q., Guo X., Taylor K., Ding J., Liu Y., Levine M., Smith J. (2021). 
            <br>
            <em>Epigenetics</em>
            <br>
            <a href="https://www.tandfonline.com/doi/abs/10.1080/15592294.2021.1939479?journalCode=kepi20">Journal</a>
    /
            <a href="https://www.medrxiv.org/content/10.1101/2021.03.01.21252660v1">Preprint</a>
            <p>
            </p>
          </td>
        </tr>

        </tr>
      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/jobloss.jpeg" alt="blind-date" width="160" height="160">
          </td>
          <td width="75%" valign="middle">
            <strong>[1]</strong><a href="https://www.nature.com/articles/s41598-021-86716-y">
              <papertitle> The impact of late-career job loss and genetic risk on body mass index: evidence from variance polygenic scores. </papertitle>
            </a>
            <br>
            Schmitz L., Goodwin J., <strong>Miao J.</strong>, Lu Q., Conley D. (2021). 
            <br>
            <em>Scientific Reports</em>
            <br>
            <a href="https://www.nature.com/articles/s41598-021-86716-y">Journal</a>
    /
            <a href="https://www.medrxiv.org/content/10.1101/2020.11.20.20235895v1">Preprint</a>
            <p>
            </p>
          </td>
        </tr>


        </tbody></table>

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
        <td style="padding:20px;width:100%;vertical-align:middle">
          <heading>Selected Awards</heading>
          <p>
            - Distinguished Student Paper Award from American Statistical Association (ASA) Section on Statistical Genomics and Genetics, 2022  [<a href="https://biostat.wiscweb.wisc.edu/2022/01/19/bds-student-jiachen-miao-wins-asa-student-paper-competition/">WiscBMI News</a>]
          </p>
          <p>
            - Reviews's Choice Award from American Society of Human Genetics Meeting (ASHG), 2021
          </p>
        </td>
      </tr>
    </tbody></table>

    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
      <tr>
      <td style="padding:20px;width:100%;vertical-align:middle">
        <heading>Software</heading>
        <p>
          - <a href="https://github.com/qlu-lab/X-Wing/">X-Wing</a> for improving genetic risk prediction in ancestrally diverse populations.  
        </p>
        <p>
          - <a href="https://github.com/qlu-lab/QUAIL/">QUAIL</a> for estimating genetic effects on the variance of quantitative traits.
        </p>
        <p>
          - <a href="https://github.com/jmiao24/SGEGG/">SGEGG</a> for fitting the linear mixed model to identify genome-wide gene-environment or gene-gene interactions for repeated measures.
        </p>
      </td>
    </tr>
  </tbody></table>

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
        <td style="padding:20px;width:100%;vertical-align:middle">
          <heading>Service</heading>
          <p>
            <strong>Reviewer</strong>
          </p>
          <p>
            - Behaviour Genetics, BMC Genomics
          </p>
          <p>
            <strong>Mentor</strong>
          </p>
          <p>
            - Gefei Song, Undergraduate Researcher at UW-Madison '22
          </p>
        </td>
      </tr>
    </tbody></table>

          




        


    <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
      <tr>
        <td>
          <br>
          <p align="right">
            <font size="2">
              <a href="https://people.eecs.berkeley.edu/~barron/">This guy makes a nice webpage.</a><br>
              Last updated in May 2022
            </font>
          </p>
        </td>
      </tr>
    </table>

      
      </td>
    </tr>
  </table>
</body>

</html>
